src/terminal/input/mouseInput.cpp

```diff
@@ -4,6 +4,7 @@
 #include "precomp.h"
 #include "terminalInput.hpp"
 
+#include "../types/inc/IInputEvent.hpp"

```

Comment:
IInputEvent lives?

Reply:
Sort of... It now contains helper functions to generate `INPUT_RECORD`s. They all start with "Synthesize".